core/src/main/scala/com/evolutiongaming/kafka/journal/FromConfigReaderResult.scala

```diff
@@ -16,7 +18,7 @@ object FromConfigReaderResult {
 
   implicit def lift[F[_]: ApplicativeThrowable]: FromConfigReaderResult[F] = {
     new FromConfigReaderResult[F] {
-      def apply[A](a: ConfigReader.Result[A]) = {
+      def apply[A: ClassTag](a: ConfigReader.Result[A]): F[A] = {
```

Comment:
why we need `ClassTag` here?

Reply:
the version of pureconfig that was published for Scala 3, some of its APIs were updated and now require ClassTag